inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/file/AbstractSource.java

```diff
@@ -128,8 +130,11 @@ public void init(InstanceProfile profile) {
         initOffset();
         registerMetric();
         initExtendHandler();
+        initSource(profile);
     }
 
+    protected abstract void initSource(InstanceProfile profile);
```

Comment:
Not all sources need `initSource`, how about  adding a default implement of here which does nothing, and sources  like `KakfaSource` override the method `initSource` if need

Reply:
The currently empty sources are because there is no real implementation, and the real source should implement this